resume/resume.org

```diff
@@ -39,10 +40,16 @@ Jacobi provides a financial modelling platform to a wide variety of businesses a
 Responsibilities
 - Build, manage and maintain Jacobi's Kubernetes infrastructure in AWS and Azure
 - Develop, improve and maintain the ansible and terraform code used to deploy that infrastructure
-- Other duties and support tasks as required.
 
 Achievements
 - Setup Gitlab CI and helped migrate software builds into it
+- Improved and expanded the company's infrastructure code
```

Comment:
In what way, specifically?

Reply:
This would be that elaboration thing I am terrible at.
I will elaborate. :)